source/Cosmos.System2_Plugs/System/Int16Impl.cs

```diff
@@ -40,20 +43,23 @@ public static Int16 Parse(string s)
 
             for (int i = z; i < s.Length; i++)
             {
-                Int16 ind = (Int16)digits.IndexOf(s[i]);
+                var ind = (short)digits.IndexOf(s[i]);
                 if (ind == -1)
                 {
-                    global::System.Console.Write("Digit '");
-                    global::System.Console.Write(s[i]);
-                    global::System.Console.WriteLine("' not found!");
+                    Console.Write("Digit '");
+                    Console.Write(s[i]);
+                    Console.WriteLine("' not found!");
```

Comment:
Better to not write on Console, we should use the debugger for this or better yet as we are throwing after to write in the exception message what went wrong.

Reply:
I just simplified the names, we should take a look at that later (#868).